data/trie/sync.go

```diff
@@ -285,10 +282,11 @@ func (ts *trieSyncer) getNode(hash []byte) (node, error) {
 
 	n, ok := ts.interceptedNodes.Get(hash)
 	if ok {
+		ts.interceptedNodes.Remove(hash)
```

Comment:
the interceptor does not delete the unused infos ?

Reply:
no, it just stores in the cache what it receives. Just like in the case of transactions, blocks and so on.

Reply:
done